wallet/src/de/schildbach/wallet/data/BlockchainIdentityBaseData.kt

```diff
@@ -30,4 +30,7 @@ data class BlockchainIdentityBaseData(val id: Int,
         get() = creationState > BlockchainIdentityData.CreationState.NONE &&
                 creationState < BlockchainIdentityData.CreationState.DONE &&
                 creationStateErrorMessage == null
+
+    val creationComplete: Boolean
+        get() = creationState >= BlockchainIdentityData.CreationState.DONE
```

Comment:
This getter could be used in other places instead of comparing to the value of DONE.

Reply:
Good idea!